tests/Rector/Property/InjectAnnotationClassRector/Fixture/inject_from_var3.php.inc

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Rector\Tests\Rector\Property\InjectAnnotationClassRector\Fixture;
+
+use Fully\Qualified\ClassName\SomeTotallyDifferentButFirstListed\Foo\Bar as FooBarFirst;
+use Fully\Qualified\ClassName\TheActual\Foo\Bar;
+use Fully\Qualified\ClassName\TheActual\Bar\Foo;
+use Fully\Qualified\ClassName\SomeTotallyDifferentButLastListed\Bar\Foo as BarFooLast;
+
+
+class InjectFromVarWithTypeOfSameName
+{
+    /**
+     * @Inject
+     *
+     * @var Bar
+     */
+    private $someBarDependency;
+
+    /**
+     * @Inject
+     *
+     * @var Foo
+     */
+    private $someFooDependency;
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Rector\Property\InjectAnnotationClassRector\Fixture;
+
+use Fully\Qualified\ClassName\SomeTotallyDifferentButFirstListed\Foo\Bar as FooBarFirst;
+use Fully\Qualified\ClassName\TheActual\Foo\Bar;
+use Fully\Qualified\ClassName\TheActual\Bar\Foo;
+use Fully\Qualified\ClassName\SomeTotallyDifferentButLastListed\Bar\Foo as BarFooLast;
+
```

Comment:
Why there is only one space now compared to 2 in original class?

Reply:
For next time, let me allow to commit to your PR, so I can merge with suggestions